cloverage/sample/cloverage/custom_reporter.clj

```diff
@@ -0,0 +1,5 @@
+(ns cloverage.custom-reporter
+  "Testing for a custom reporter")
+
+(defn reporter-fn [{:keys [:args]}]
```

Comment:
The custom reporter function for the tests.  Just put out of the way in a separate namespace.